doc/user-guide/interactive.md

```diff
@@ -0,0 +1,115 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Interactive SQL Queries
```

Comment:
~~~suggestion
# Interactive SQL Queries

```{versionadded} 0.7
```

~~~

Reply:
Fix in [82184f6](https://github.com/ploomber/jupysql/pull/293/commits/82184f6bdc7731a290463ff834731efb4d1af651)